File: Examples/VisualBasic/AsposePDF/Links-Actions/RemoveOpenAction.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf
Namespace AsposePDF.LinksActions
    Public Class RemoveOpenAction
        Public Shared Sub Run()
            ' ExStart:RemoveOpenAction
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_LinksActions()
            ' Open document
            Dim document As New Document(dataDir & Convert.ToString("RemoveOpenAction.pdf"))
            ' Remove document open action
            document.OpenAction = Nothing
            dataDir = dataDir & Convert.ToString("RemoveOpenAction_out_.pdf")
            ' Save updated document
            document.Save(dataDir)
            ' ExEnd:RemoveOpenAction
            Console.WriteLine(Convert.ToString(vbLf & "Open action removed successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePDF/Links-Actions/RemoveOpenAction.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.LinksActions
{
    public class RemoveOpenAction
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_LinksActions();

            //open document
            Document document = new Document(dataDir + "RemoveOpenAction.pdf");

            //remove document open action
            document.OpenAction = null;

            //save updated document
            document.Save(dataDir + "RemoveOpenAction_out.pdf");
            
            
        }
    }
}